plugins/pantheon-files-ctags/plugin.vala

```diff
@@ -210,26 +190,25 @@ public class Marlin.Plugins.CTags : Marlin.Plugins.Base {
         if (file.get_ftype () == "application/octet-stream") {
             unknowns.push_head (file);
 
-            if (idle_consume_unknowns == 0)
+            if (idle_consume_unknowns == 0) {
                 idle_consume_unknowns = Idle.add (() => {
                                                   consume_unknowns_queue.begin ();
                                                   idle_consume_unknowns = 0;
                                                   return false;
                                                   });
+            }
         }
     }
 
     private async void rreal_update_file_info (GOF.File file) {
         return_if_fail (file != null);
-
+warning ("rreal update file info %s", file.uri);
```

Comment:
Fix identation here and don't use `warning ()` here, it's better to use something like `debug ()`.